connectors.go

```diff
@@ -86,6 +86,13 @@ type AuthMetadataConnector interface {
 	GetPostAuthInfo(ctx context.Context) (*common.PostAuthInfo, error)
 }
 
+type WebhookVerifiableConnector interface {
```

Comment:
I think `WebhookVerifierConnector` makes more sense since the connector is the one verifying the message